spec/features/admin/shelters/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe "Admin Shelter Index page", type: :feature do
+  before :each do
+    @shelter_1 = Shelter.create!(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+    @shelter_2 = Shelter.create!(name: "Mystery Building", city: "Irvine, CA", foster_program: false, rank: 9)
+    @shelter_3 = Shelter.create!(name: "Dumb Friends League", city: "Denver, CO", foster_program: true, rank: 10)
+  end
+
+  describe "As a visitor" do
+    describe "When I visit the admin shelter index ('/admin/shelters')" do
+
+      # 10. Admin Shelters Index
+      it "Then I see all Shelters in the system listed in reverse alphabetical order by name" do
+        visit "/admin/shelters"
+
+        within ("#all-shelters") do
```

Comment:
Nice, this is referencing the div tag.  I like it.

Reply:
TBH, it helps when you look at how the existing functionality of the pets, shelters, and vets pages are set up. There's a lot of good existing code there.